core/tools/zenoh/bootstrap.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Immediately exit on errors
+set -e
+
+PROJECT_NAME="zenoh"
+VERSION="1.0.0"
+BINARIES=(
+  "zenoh"
+  "zenoh-plugin-webserver"
+  "zenoh-backend-filesystem"
+)
+
+echo "Installing project $PROJECT_NAME and friends with version $VERSION"
+
+# Step 1: Prepare the download URL
+
+ARCH="$(uname -m)"
+case "$ARCH" in
+  x86_64 | amd64)
+    TOOLCHAIN="x86_64-unknown-linux-musl"
+    ;;
+  armv7l | armhf)
+    TOOLCHAIN="armv7-unknown-linux-gnueabihf"
+    ;;
+  aarch64 | arm64)
+    TOOLCHAIN="aarch64-unknown-linux-musl"
+    ;;
+  *)
+    echo "Architecture: $ARCH is unsupported, please create a new issue on https://github.com/bluerobotics/BlueOS/issues"
+    exit 1
+    ;;
+esac
+echo "For architecture $ARCH, using toolchain $TOOLCHAIN"
+
+# Step 2: Prepare the installation and tools paths
+
+if [ -n "$VIRTUAL_ENV" ]; then
+    BIN_DIR="$VIRTUAL_ENV/bin"
+else
+    BIN_DIR="/usr/bin"
+fi
+
+BINARY_PATH="$BIN_DIR/$PROJECT_NAME"
```

Comment:
I think we actually want this instead:

```suggestion
BINARY_PATH="$BIN_DIR/"
```

otherwise we'd have to add that to the PATH